src/sknnr_spatial/datasets/_base.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import annotations
+
+from importlib import resources
+
+import numpy as np
+import pandas as pd
+import rasterio
+import rioxarray
+import sknnr.datasets
+import xarray as xr
+from numpy.typing import NDArray
+
+DATA_MODULE = "sknnr_spatial.datasets.data"
+
+
+def _load_rasters_to_dataset(
+    file_names: list[str], *, var_names: list[str], module_name: str, chunks=None
+) -> xr.Dataset:
+    """Load a list of rasters from the data module as an xarray Dataset."""
+    das = []
+    for file_name, var_name in zip(file_names, var_names):
+        bin = resources.open_binary(module_name, file_name)
+
+        da = rioxarray.open_rasterio(bin, chunks=chunks)
+        da = da.to_dataset(dim="band").rename({1: var_name})
+        das.append(da)
+
+    return xr.merge(das)
+
+
+def _load_rasters_to_array(file_names: list[str], *, module_name: str) -> np.ndarray:
+    """Load a list of rasters from the data module as a numpy array."""
+    arr = None
+    for file_name in file_names:
+        bin = resources.open_binary(module_name, file_name)
+
+        with rasterio.open(bin) as src:
+            band = src.read(1)
+            arr = band if arr is None else np.dstack((arr, band))
+
+    return arr
+
+
+def load_swo_ecoplot(
+    as_dataset: bool = False,
+) -> tuple[NDArray | xr.Dataset, pd.DataFrame, pd.DataFrame]:
+    """Load the southwest Oregon (SWO) USFS Region 6 Ecoplot dataset.
+
+    The dataset contains:
+
+     1. **Image data**: 128x128 pixel GeoTIFF image chips of 18 environmental and
+        spectral variables at 30m resolution.
+     2. **Plot data**: 3,005 plots with environmental, Landsat, and forest cover
+        measurements. Ocular measurements of tree cover (COV) are categorized by
+        major tree species present in southwest Oregon.  All data were collected in 2000
+        and Landsat imagery processed through the CCDC algorithm was extracted for the
```

Comment:
What do you think about adding a reference to CCDC?  It would be:

```
Zhu Z, CE Woodcock, P Olofsson. 2012. Continuous monitoring of forest disturbance using all available Landsat imagery. Remote Sensing of Environment. 122:75–91. 
```

Reply:
Good idea, I'll add that!